### PR TITLE
fix: Removed MCDU Link button from EFB to avoid user confusion

### PR DESCRIPTION
**Pull request by frankkopp**
## Summary of Changes

As we regularly have users reporting in #support that the MCDU Link button on the EFB Dashboard does not work or users being confused by it I want to propose to remove it. 

As it has currently no function at all it doesn't make any sense anyway. 

I only have commented out the section and centered the "FROM SIMBRIEF" button.
Easy to undo later if the button shall get functionality. 

## Screenshots (if necessary)
image

## References
N/A

Discord username (if different from GitHub): Cdr_Maverick

## Testing instructions
There should not be anything to test as the button was already deactivated. 
Making sure the EFB still works as expected can't hurt though. 

## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page

### CODE REVIEW

**Comment by GhostEagle68** [comment]
Instead of removing it, how about just renaming to something like "Coming Soon"? 

OR if you remove it the simbreif button should be centered at least. Looks off aligned to the left 🙂

**Comment by frankkopp** [comment]
> Instead of removing it, how about just renaming to something like "Coming Soon"?
> 
> OR if you remove it the simbreif button should be centered at least. Looks off aligned to the left 🙂

I did not change too much and mess something else up. 
Coming soon also doesn't make a lot of sense as it would be unclear what comes soon - the button function itself is not clear :)

I'm have not much practice with that stuff - happy to try to center it - it would be nicer, I agree.

**Comment by GhostEagle68** [comment]
> > Instead of removing it, how about just renaming to something like "Coming Soon"?
> > 
> > OR if you remove it the simbreif button should be centered at least. Looks off aligned to the left 🙂
> 
> I did not change too much and mess something else up. 
> Coming soon also doesn't make a lot of sense as it would be unclear what comes soon - the button function itself is not clear :)
> 
> I'm have not much practice with that stuff - happy to try to center it - it would be nicer, I agree. 
> 

Yea you may want to try and mess with margin-left. Should work out fine. Can't recall where it goes exactly though

**Comment by frankkopp** [comment]
> > > Instead of removing it, how about just renaming to something like "Coming Soon"?
> > > OR if you remove it the simbreif button should be centered at least. Looks off aligned to the left 🙂
> > 
> > 
> > I did not change too much and mess something else up.
> > Coming soon also doesn't make a lot of sense as it would be unclear what comes soon - the button function itself is not clear :)
> > I'm have not much practice with that stuff - happy to try to center it - it would be nicer, I agree.
> 
> Yea you may want to try and mess with margin-left. Should work out fine. Can't recall where it goes exactly though

FROM SIMBRIEF button is centered now.

**Comment by GhostEagle68** [comment]
> > > > Instead of removing it, how about just renaming to something like "Coming Soon"?
> > > > OR if you remove it the simbreif button should be centered at least. Looks off aligned to the left 🙂
> > > 
> > > 
> > > I did not change too much and mess something else up.
> > > Coming soon also doesn't make a lot of sense as it would be unclear what comes soon - the button function itself is not clear :)
> > > I'm have not much practice with that stuff - happy to try to center it - it would be nicer, I agree.
> > 
> > Yea you may want to try and mess with margin-left. Should work out fine. Can't recall where it goes exactly though
> 
> FROM SIMBRIEF button is centered now. 

LGTM!

**Comment by tracernz** [comment]
I think this is a good change, and we should generally avoid adding UI things that don't work in future. 

You will need to address these lint errors. You could simply delete the button entirely if you like rather than commenting it out. git is very capable of retaining the history if somebody wants it back one day.
```
/external/src/instruments/src/EFB/Dashboard/Widgets/FlightWidget.tsx
    4:5   error  'IconLink' is defined but never used. Allowed unused vars must match /^_/u  @typescript-eslint/no-unused-vars
  136:26  error  Expected exception block, space or tab after '/*' in comment                spaced-comment
  136:26  error  Expected space or tab before '*/' in comment                                spaced-comment
  137:26  error  Expected space or tab before '*/' in comment                                spaced-comment
  138:26  error  Expected space or tab before '*/' in comment                                spaced-comment
  139:26  error  Expected exception block, space or tab after '/*' in comment                spaced-comment
  139:26  error  Expected space or tab before '*/' in comment                                spaced-comment
  140:26  error  Expected space or tab before '*/' in comment                                spaced-comment
  141:26  error  Expected space or tab before '*/' in comment                                spaced-comment
  142:26  error  Expected space or tab before '*/' in comment                                spaced-comment
  143:26  error  Expected exception block, space or tab after '/*' in comment                spaced-comment
  143:26  error  Expected space or tab before '*/' in comment                                spaced-comment
  ```

File: src/instruments/src/EFB/Dashboard/Widgets/FlightWidget.tsx [hunk 1]
```diff
@@ -133,14 +133,14 @@ const FlightWidget = (props: FlightWidgetProps) => {
...
-                            LINK MCDU
-                        </button>
+                        {/*<button*/}
+                        {/*    type="button"*/}
+                        {/*    className="ml-1 w-1/2 text-white bg-green-500 p-2 flex items-center justify-center rounded-lg focus:outline-none opacity-50 text-lg"*/}
+                        {/*>*/}
+                        {/*    <IconLink className="mr-2" size={23} stroke={1.5} strokeLinejoin="miter" />*/}
+                        {/*    {' '}*/}
+                        {/*    LINK MCDU*/}
+                        {/*</button>*/}
```
Comment (bouveng) [inline src/instruments/src/EFB/Dashboard/Widgets/FlightWidget.tsx:143]:
```suggestion

```
Hi man! Just get rid of it. :see_no_evil:

**Review by bouveng — CHANGES_REQUESTED** [review]

File: src/instruments/src/EFB/Dashboard/Widgets/FlightWidget.tsx (hunk 1 above)
Comment (frankkopp) [reply src/instruments/src/EFB/Dashboard/Widgets/FlightWidget.tsx:143]: Will do :D

**Review by bouveng — APPROVED** [review]

**Comment by frankkopp** [comment]
> /external/src/instruments/src/EFB/Dashboard/Widgets/FlightWidget.tsx
>     4:5   error  'IconLink' is defined but never used. Allowed unused vars must match /^_/u  @typescript-eslint/no-unused-vars

got rid of this as well

**Review by bouveng — APPROVED** [review]

**Comment by marcman86** [comment]
Quality Assurance Tester Report
Discord:marcman86#4907
Object of testing: #6148
Tier of Testing: 1
Date: 7/11/2021

Testing Process:
- Spawned C&D at EDDP
- Turned on bat., ext. power and EFB
- Tested "FROM SIMBRIEF" button (Image1)


Negatives: - 

Testing Results: Passed
Conclusions: Works for me. Is it possible to change the color of the button? White on cyan is hard to read. 

Hope this helps :-)

Media:


Image1

**Comment by frankkopp** [comment]
> Testing Results: Passed Conclusions: Works for me. Is it possible to change the color of the button? White on cyan is hard to read.

I agree. I did not want to change that as I'm not a main dev for this. Just wanted to remove the other button.

I can test a few different colors and create a new PR for it. If others agree we can change the color then.

**Pull request merged**